trading-hello-service-api/build.sbt

```diff
@@ -0,0 +1,13 @@
+// Nexus configuration
+publishTo := {
+  val nexus = "https://nexus3.softwaremill.com/"
```

Comment:
You can also add `repository/` to the Nexus URL, since it's common for both snapshots and releases.